### PR TITLE
parse compiler output for errors/warnings and make it clickable to jump ...

### PR DESCRIPTION
**Pull request by Extrawurst**
...to sourcefile/line

this fixes issue #15

### CODE REVIEW

**Comment by Extrawurst** [comment]
travisCI fails cause of issues in the last dlib version tag 0.4.1 with unittest code that was already fixed but not tagged again:
https://github.com/gecko0307/dlib/issues/47

File: src/dlangide/ui/frame.d [hunk 1]
```diff
@@ -279,6 +298,7 @@ class IDEFrame : AppFrame {
         _dockHost.addDockedWindow(_wsPanel);
 
         _logPanel = new OutputPanel("output");
+		_logPanel.compilerLogIssueClickHandler = &onCompilerLogIssueClick;
```
Comment (Freakazo) [inline src/dlangide/ui/frame.d]: Weird indent?

**Comment by buggins** [comment]
Thank you!

**Comment by buggins** [comment]
Working great :) TODO: Highlight of error and warning lines for better usability.

**Pull request merged**